contracts/contracts/NoLossDao.sol

```diff
@@ -134,6 +134,14 @@ contract NoLossDao is Initializable {
     );
     _;
   }
+
+  modifier lockInFulfilled(address givenAddress) {
```

Comment:
Totally happy with this. We should keep in mind a withdraw period too potential. I can think of pros and cons so not clear cut.